packages/firestore/src/remote/remote_store.ts

```diff
@@ -585,10 +585,7 @@ export class RemoteStore implements TargetMetadataProvider {
 
   private tryRecoverFromPrimaryLeaseLoss(err: FirestoreError): void {
```

Comment:
Please rename this if you rename the other one.

Reply:
Done